modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -45,33 +44,31 @@ Labels on a virtual machine are passed through to the pod. The `special: key` la
 apiVersion: v1
 kind: Service
 metadata:
-  name: vmservice <1>
-  namespace: example-namespace <2>
+  name: example_service
+  namespace: example_namespace
 spec:
-  externalTrafficPolicy: Cluster <3>
+  externalTrafficPolicy: Cluster <1>
```

Comment:
I would just delete this. The example should show a basic Service and how to tie it to a VM. Low level attributes such as this one should be documented in OCP's docs dedicated to Services